.travis.yml

```diff
@@ -10,7 +10,7 @@ before_install:
   - export PATH=${HOME}:${PATH}
   - export OPAMYES=1
   - export OPAMJOBS=2
-  - opam init --compiler=${OCAML_VERSION}
+  - opam init --compiler=${OCAML_VERSION} --disable-sandboxing
```

Comment:
Just FMI, do we do something ill-advised in the ocamlformat build/test scripts that needs this?

Reply:
I think I see now. Fixed in ac68d057167831264d981092b108f9cf3f275c59.

Reply:
AFAIK, this is needed because travis already use a sandbox (in that case Docker I think) and bublewrap doesn't support nested sandboxing.